File: core/src/main/java/com/ortin/gesturetranslator/core/managers/mediapipe/listeners/MPDetectionListener.java
package com.ortin.gesturetranslator.core.managers.mediapipe.listeners;

import com.ortin.gesturetranslator.core.managers.mediapipe.models.MPDetection;

import io.reactivex.rxjava3.annotations.NonNull;

public interface MPDetectionListener {
    void detect(MPDetection mpDetection);

    void error(Exception exception);
}

File: core/src/main/kotlin/com/ortin/gesturetranslator/core/managers/mediapipe/listeners/MPDetectionListener.kt
package com.ortin.gesturetranslator.core.managers.mediapipe.listeners

import com.ortin.gesturetranslator.core.managers.mediapipe.models.MPImageDetection

interface MPDetectionListener {
    fun onResults(mpImageDetection: MPImageDetection?)
    fun onError(exception: Exception)
}
